### PR TITLE
Documentation/rw readme

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
README has been updated with badges, table of contents, fleshed out future directions section, back end repo & gem links, external libraries added to tech stack, and more!

**Note: Gifs still need to be added. This will be done when final style/design branch is pushed up!** 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Other, Please explain: **Documentation**
## Has This Been Tested?
- [X] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:   
No test needed for a README!

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,77 @@
+[![Build Status](https://travis-ci.org/SOM-Timer/som_timer_fe.svg?branch=main)](https://travis-ci.org/SOM-Timer/som_timer_fe)
+[![Contributors][contributors-shield]][contributors-url]
+[![Issues][issues-shield]][issues-url]
+[![Issues][issues-closed-shield]][issues-closed-url]
+
```
Comment (Abdeboskey) [inline README.md:5]: All the badges! So profresh.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,77 @@
...
 # Som Timer
-Som Timer is a timer that cares. Based on the pomodoro technique™️, this application provides users with curated wellness content during break intervals. Build in rest, so you can focus best.
+
+Som Timer is a timer that cares. Based on the Pomodoro technique™️, this application provides users with curated wellness content during break intervals. Build in rest, so you can focus best.
```
Comment (Abdeboskey) [inline README.md:8]: `™` !

**Review by Abdeboskey — APPROVED** [review]
This is one healthy lookin readme. I love the badges, the TOC, the future iterations, all really nicely done. Thank you for your work on this Rachel!!

File: README.md [hunk 3]
```diff
@@ -1,36 +1,77 @@
+[![Build Status](https://travis-ci.org/SOM-Timer/som_timer_fe.svg?branch=main)](https://travis-ci.org/SOM-Timer/som_timer_fe)
+[![Contributors][contributors-shield]][contributors-url]
+[![Issues][issues-shield]][issues-url]
+[![Issues][issues-closed-shield]][issues-closed-url]
+
 # Som Timer
```
Comment (Abdeboskey) [inline README.md:6]: What if we put in 
```markdown
<div align="center"><img src="src/assets/content/meditationContentIcon.png" alt="woman sitting in meditation pose"/>
<img src="src/assets/content/somaticContentIcon.png" alt="human head with arrows connecting to body"/>
<img src="src/assets/content/yogaContentIcon.png" alt="woman in warrior yoga pose"/></div>

 # <div align="center">Som Timer</div>
```

where the Title is?

Comment (rwilliams659) [reply README.md:6]: I tried popping those in there & adding a height of 30px to each image. What do you think? 
Screen Shot 2020-11-03 at 10 08 35 PM

Comment (jkwest-93) [reply README.md:6]: I love this!!!! Adding the icons is SO the move!

Will the line colors turn dark on a light background? I think GitHub default is white!

**Review by jkwest-93 — APPROVED** [review]
Thanks so much for updating this! This sounds great & I love the addition of the icons.

File: README.md [hunk 4]
```diff
@@ -1,36 +1,77 @@
...
+## Future Directions
+This current version of the app is just the beginning, and we would like to significantly expand our scope. The following are our plans for future iterations: 
+#### Add Users
+  - As this app was initially built in just 2 weeks, the demo does not include unique users. We would like to add the ability to create users, complete with sign-up and log-in screens. 
+#### Expand customization
+  - We want to ensure that Som Timer works well for all users, so we would like to expand our customization options on the Settings page. Current ideas for this include adding the ability to disable mood ratings, toggle between themes, such as light and dark mode, and select the categories of content the user can then choose from during breaks. 
+#### Allow a user to upload content
+  - In the vein of expanding customization, we would also like to give the user the ability to add new content to their personal library by uploading links from YouTube, Vimeo, SoundCloud, and other platforms. 
+
+## Back End Information
```
Comment (jkwest-93) [inline README.md:54]: Thanks for adding this section!!!

**Review by jkwest-93 — APPROVED** [review]

**Pull request merged**